### ISSUE TITLE
[Enhancement] Dark theme

### ISSUE THREAD

**Issue by ParticleMon** [munt/munt#93]
The floating LCD would look even better if its window border were removed and the background of the MIDI MESSAGE indicator matched the black case of the MT-32.

A bonus would be if mt32emu_qt had a dark theme option.

**Comment by sergm** [comment]
I guess, you looked at it on Cinnamon desktop, right? Then I am not sure I get the request, because that seems to be already implemented (as can be seen on the screenshot attached to #94).

AFAICT, Qt applications rely on the system theme by default, so adding just a dark custom theme would be a bit strange. I don't mind adding support for setting a configurable style though, but definitely not limited to just dark one.

**Comment by ParticleMon** [comment]
At least for me, with a dark theme in Cinnamon, Munt has a light theme.

**Comment by sergm** [comment]
Right, for me too as long as I am using the bundled appimage. Native build is not isolated from the DE and uses the system default style, I assume. But I am not too involved into Qt internals, so may be wrong.

**Comment by ParticleMon** [comment]
Gotcha. I use AppImages and the like only when necessary, and unfortunately, mt32emu_qt has been one such case.

I would prefer to use a native build, but at least for this non-developer, how to build mt32emu_qt is unclear. The instructions to build mt32emu described to "probably" work do work for me, but now what? Steps, not just vague hints, would be very much appreciated, surely also by others.

Considered providing a .deb? (I would understand that if you were to, then others might request a .rpm and so forth, which may be more than you wish to do.)

**Comment by sergm** [comment]
Oh well, the instructions really seem unclear 😕 
OTOH, you might find [the pre-built binary packages](https://sourceforge.net/projects/munt/files/munt/2.6.2/Linux/) useful. Also, a few GNU distributions have mt32emu library incorporated, so worth having a look around.

**Comment by ParticleMon** [comment]
For some forgotten reason, I have been usnig snapshots, of which pre-built binaries have not been provided.

Since a .deb works (at least now), this issue is solved.

But this leads to a question: Why not post all releases here?
